Cargo.toml

```diff
@@ -17,6 +17,7 @@ default = ["http_server", "rand", "uuid", "tracing-span-filter"]
 hyper = ["dep:hyper", "http-body-util", "restate-sdk-shared-core/http"]
 http_server = ["hyper", "hyper/server", "hyper/http2", "hyper-util", "tokio/net", "tokio/signal", "tokio/macros"]
 tracing-span-filter = ["dep:tracing-subscriber"]
+schemars = ["dep:schemars"]
```

Comment:
I wonder if this should be something enabled by default, as we do in the other SDKs.

But if we do so, then i wonder if we should re-export (ofc behind feature gate) the `JsonSchema` derive in the prelude, such that in default features case you got everything you need without importing another crate... WDYT?